website/docs/reference/global-configs.md

```diff
@@ -3,49 +3,65 @@ title: "Global Configs"
 id: "global-configs"
 ---
 
-dbt supports several global runtime configs. Starting in v1.0, all these configs can be set in three places:
-- `config:` block in `profiles.yml`, sometimes called "user config"
-- environment variables, prefixed with `DBT_`
-- CLI flags, which immediately follow `dbt` and precede your subcommand
+### About Global Configs
 
-Notes:
-- The precedence order is always (1) CLI flag > (2) env var > (3) profile config
-- All boolean configs can be turned on via a CLI flag named `--this-config`, and turned off with a CLI flag named `--no-this-config`
-- Use the profile config to set defaults for all projects running on your local machine
+Global configs enable you to fine-tune how dbt runs projects on your machine—whether your personal laptop, an orchestration tool running remotely, or (in some cases) dbt Cloud. They differ from [project configs](reference/dbt_project.yml) and [resource configs](reference/configs-and-properties), which tell dbt _what_ to run.
 
-<File name='profiles.yml'>
+Global configs control things like the visual output of logs, the manner in which dbt parses your project, and what to do when dbt finds a version mismatch or a failing model.
 
-```yaml
+These configs are "global" because they are available for all dbt commands, and because they apply across all projects run on the same machine.
 
-config:
-  this_config: true
+Starting in v1.0, you can set global configs in three places. When all three are set, command line flags take precedence, then environment variables, and last profile configs.
+
+#### Command line flags
+
+Command line (CLI) flags immediately follow `dbt` and precede your subcommand. When set, CLI flags override environment variables and profile configs. You can use `--this-config` CLI flag to turn on boolean configs, and a `--no-this-config` CLI flag to turn off boolean configs:
```

Comment:
```suggestion
Command line (CLI) flags immediately follow `dbt` and precede your subcommand. When set, CLI flags override environment variables and profile configs. 

Non-boolean config structure: 

```text
$ --<THIS-CONFIG>=<SETTING> <SUBCOMMAND>
```

Non-boolean config example:

```text
$ dbt --printer-width=80 run
$ dbt --indirect-selection=eager test
```

To turn on boolean configs, you would use the `--<THIS-CONFIG>` CLI flag, and a `--no-<THIS-CONFIG>` CLI flag to turn off boolean configs, replacing <THIS-CONFIG> with the config you are enabling or disabling and <SUBCOMMAND>  with the command this config applies to.
```